### PR TITLE
Added new fractal code

### PR DESCRIPTION
**Pull request by rudransh61**
### Describe your change:
I have added a new fractal code in turtle .

[x] Add an algorithm?

### Checklist:
[x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
[x] This pull request is all my own work -- I have not plagiarized.
[x] I know that pull requests will not be merged if they fail the automated tests.
[x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
[x] All new Python files are placed inside an existing directory.
[x] All filenames are in all lowercase characters with no spaces or dashes.
[x] All functions and variable names follow Python naming conventions.
[x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
[x] All new algorithms include at least one URL that points to Wikipedia or another similar explanation.

### CODE REVIEW

**Comment by rudransh61** [comment]
sir is there any issue or an problem ???
pls inform

**Comment by rudransh61** [comment]
sir @cclauss  idk why it is showing build ruff tests fail ?? idk how to debug them?????

**Comment by cclauss** [comment]
```
pip install ruff
ruff --fix --select=I fractals/tree_fractal.py
```

File: fractals/tree_fractal.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
+"""
+Author : Rudransh Bhardwaj
+Github : @rudransh61
+Wikipedia to learn about fractal tree (or canopy) : https://en.wikipedia.org/wiki/Fractal_canopy
+"""
+
+import turtle  # importing the module required
+
+
```
Comment (cclauss) [inline fractals/tree_fractal.py:9]:
```suggestion
```

**Comment by cclauss** [comment]
This crashes on macOS and seems to crash in our continuous integration builds.  What operating system did you test this on?

**Comment by cclauss** [comment]
mypy.....................................................................Failed
- hook id: mypy
- exit code: 1

fractals/tree_fractal.py:48: error: Argument 1 to "tree" has incompatible type "float"; expected "int"  [arg-type]
fractals/tree_fractal.py:56: error: Argument 1 to "tree" has incompatible type "float"; expected "int"  [arg-type]
Found 2 errors in 1 file (checked 1205 source files)

**Comment by rudransh61** [comment]
Windows..

**Pull request closed without merging**